code/game/machinery/computer/security.dm

```diff
@@ -570,6 +469,18 @@
 
 	..(severity)
 
+/**
+  * Sets a temporary message to display to the user
+  *
+  * Arguments:
+  * * text - Text to display, null/empty to clear the message from the UI
+  * * style - The style of the message: (color name), info, success, warning, danger
+  */
+/obj/machinery/computer/secure_data/proc/set_temp(text = "", style = "info", update_now = FALSE)
```

Comment:
Wouldnt you want to default this to `TRUE`

Reply:
Not really, this proc is mostly called in `tgui_act` which already handles the refresh